tools/report-converter/tests/unit/analyzers/ubsan_output_test_files/ubsan1_nonmatching_msg.plist

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>diagnostics</key>
+	<array>
+		<dict>
+			<key>category</key>
+			<string>unknown</string>
+			<key>check_name</key>
+			<string>UndefinedBehaviorSanitizer</string>
```

Comment:
How can this be that, if it was supposedly initialised to `None`?

Reply:
The idea is that when `checker_name` is `None`, we fall back to a default checker name later down the road, which is the name of the sanitizer.